README.md

```diff
@@ -30,32 +30,28 @@ from ionic_langchain.tool import IonicTool
 from langchain.agents import AgentType, Tool
 from langchain.agents import initialize_agent
 from langchain.chat_models import ChatOpenAI
-from langchain.memory import ConversationBufferWindowMemory, RedisChatMessageHistory
 
-
-tools: List[Tool] =  [
+tools =  [
     IonicTool().tool(),
-    # others,
+    # your other tools,
 ]
-redis_memory = RedisChatMessageHistory(url=os.environ.get("REDIS_URL"),session_id="chatId"),
-memory = ConversationBufferWindowMemory(
-    k=12,
-    return_messages=True,
-    chat_memory=redis_memory,
-    memory_key="chat_history",
-)
 
 agent = initialize_agent(
     tools=tools,
-    llm=ChatOpenAI(openai_api_key=os.environ.get("OPENAI_API_KEY"),temperature=0.5),
-    agent=AgentType.CHAT_CONVERSATIONAL_REACT_DESCRIPTION,
-    memory=memory,
+    llm=ChatOpenAI(openai_api_key="your_key_here", temperature=0.7),
```

Comment:
should we change our impl?

Reply:
in our agent? no that works because it has redis, etc.